src/handlers/shared/check-assignments.ts

```diff
@@ -0,0 +1,59 @@
+import { Context, IssueEvent } from "../../types";
+
+const UBIQUIBOT_ID = 113181824;
```

Comment:
What does this value correspond to, the `APP_ID`? I think this should not be hardocded.

Reply:
https://github.com/ubiquibot/command-start-stop/pull/19#discussion_r1696328279

If it's configurable that allows partners to decide which by @0x4007's comments here that doesn't sound right

Reply:
Got it, but then when we use the plugin within another organization won't this be broken?

Reply:
as far as I understand `user.id` will always be the same for the production Ubiquibot and won't ever change but it's name can change etc and `installID` would be different for each or

Reply:
Then that would be fine I believe. My concern is when we are installing in a different organization so it doesn't break there, such as when we run our own tests.

Reply:
Ah I see yeah, for testing on V2 this wouldn't work. Perhaps I can hardcode our production and development botIDs and have a config item to use the dev bot?